x/evmutil/keeper/keeper.go

```diff
@@ -20,6 +20,10 @@ type Keeper struct {
 	bankKeeper    types.BankKeeper
 	evmKeeper     types.EvmKeeper
 	accountKeeper types.AccountKeeper
+
+	// the address capable of executing a MsgEVMCall message. Typically, this
+	// should be the x/community module account.
+	authority string
```

Comment:
making this a slice now could be useful for future proofing if / when we want to expand the list of authorities